plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergPageSinkProvider.java

```diff
@@ -74,10 +76,11 @@ private ConnectorPageSink createPageSink(ConnectorSession session, IcebergWritab
         HdfsContext hdfsContext = new HdfsContext(session);
         Schema schema = SchemaParser.fromJson(tableHandle.getSchemaAsJson());
         PartitionSpec partitionSpec = PartitionSpecParser.fromJson(schema, tableHandle.getPartitionSpecAsJson());
+        LocationProvider locationProvider = LocationProviders.locationsFor(tableHandle.getOutputPath(), tableHandle.getProperties());
         return new IcebergPageSink(
                 schema,
                 partitionSpec,
-                tableHandle.getOutputPath(),
+                locationProvider,
```

Comment:
doesn't seem to belong to "pass in table properties to initialize location provider in sink prov…" commit